io.github.chkr1011.mqttMultimeter.metainfo.xml

```diff
@@ -0,0 +1,87 @@
+  <project_license>MIT</project_license>
+
+  <developer_name>Christian Kratky</developer_name>
+
+  <url type="homepage">https://github.com/chkr1011/mqttMultimeter</url>
+
+  <content_rating type="oars-1.0">
+    <content_attribute id="violence-cartoon">none</content_attribute>
+    <content_attribute id="violence-fantasy">none</content_attribute>
+    <content_attribute id="violence-realistic">none</content_attribute>
+    <content_attribute id="violence-bloodshed">none</content_attribute>
+    <content_attribute id="violence-sexual">none</content_attribute>
+    <content_attribute id="drugs-alcohol">none</content_attribute>
+    <content_attribute id="drugs-narcotics">none</content_attribute>
+    <content_attribute id="drugs-tobacco">none</content_attribute>
+    <content_attribute id="sex-nudity">none</content_attribute>
+    <content_attribute id="sex-themes">none</content_attribute>
+    <content_attribute id="language-profanity">none</content_attribute>
+    <content_attribute id="language-humor">none</content_attribute>
+    <content_attribute id="language-discrimination">none</content_attribute>
+    <content_attribute id="social-chat">none</content_attribute>
+    <content_attribute id="social-info">none</content_attribute>
+    <content_attribute id="social-audio">none</content_attribute>
+    <content_attribute id="social-location">none</content_attribute>
+    <content_attribute id="social-contacts">none</content_attribute>
+    <content_attribute id="money-purchasing">none</content_attribute>
+    <content_attribute id="money-gambling">none</content_attribute>
+  </content_rating>
+
+  <releases>
+    <release version="1.9.0.290" date="2024-08-04">
+      <description>https://github.com/chkr1011/mqttMultimeter/releases/tag/v1.9.0.290</description>
+    </release>
+  </releases>
+
+  <recommends>
+    <control>pointing</control>
+    <control>keyboard</control>
+  </recommends>
+
+  <description>
+<p>
+MQTT Multimeter is a cross platform .NET application for MQTT communication testing and analysing.
+</p>
+
+<h1>Features</h1>
+<p>
+<ul>
+<li>Powerful inspection of message data.</li>
+<li>Inspection of raw MQTT messages.</li>
+<li>Publishing of messages.</li>
+<li>Subscribing of messages.</li>
+<li>MQTTv5 features like user properties etc.</li>
+<li>Storage of application state (connections, subscriptions etc.).</li>
+</ul>
+</p>
+
+<p>
+The protocol implementation is based on the MQTTnet project. The application uses Avalonia for UI rendering and runs on nearly any platform and CPU architecture.
+</p>
+
+<h1>Supported MQTT protocol versions</h1>
+<p>
+<ul>
+    <li>3.1.0</li>
+    <li>3.1.1</li>
+    <li>5.0.0</li>
+</ul>
+</p>
+
+  </description>
+
+  <launchable type="desktop-id">io.github.chkr1011.mqttMultimeter.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/chkr1011/mqttMultimeter/main/Images/Screenshots/InflightPage.png</image>
```

Comment:
linking to a branch will result in breakage. you should link to a specific tag or commit instead.